src/differentiation_interface.jl

```diff
@@ -64,9 +101,12 @@ macro define_with_derivatives(f, df, ddf)
         local g = $f
         RigorousInvariantMeasures.value_and_derivative(::typeof(g), x) = ($f(x), $df(x))
         RigorousInvariantMeasures.value_derivative_and_second_derivative(::typeof(g), x) = ($f(x), $df(x), $ddf(x))
+        RigorousInvariantMeasures.derivative(::typeof(g), x) = $df(x)
+        RigorousInvariantMeasures.second_derivative(::typeof(g), x) = $ddf(x)
+        RigorousInvariantMeasures.derivative_and_second_derivative(::typeof(g), x) = ($df(x), $ddf(x))
         g
     end
 end
 
-
-export value_and_derivative, value_derivative_and_second_derivative, @define_with_derivatives
+export value_and_derivative, value_derivative_and_second_derivative, derivative, 
```

Comment:
Maybe it is better to put the exported methods at the top of the file, which makes easier to identify which file is exporting what.
Probably it does not make such a big difference, but anyway, if it is possible I would keep it like that.